src/rest/Query.ts

```diff
@@ -15,6 +15,16 @@ export interface IFacetOptions {
   freezeFacetOrder?: Boolean;
 }
 
+/**
+ * The information about the user we'd like to check and its actions.
+ */
+export interface IUserActions {
+  /**
+   * The id of the user for which we should check the document views.
+   */
+  tagViewsOfUser?: string;
```

Comment:
Is it really optional from the search API ? 

Meaning, if you add a userAction query param, I'm guessing this value then become mandatory or you get a bad request ?

Reply:
Yes, indeed, from my understanding you should not have a userAction in the result that does not include tagViewsOfUser. I am guessing however that might change if other fields ends up in userActions, but then `IUserActions` can be updated when this will come.

Reply:
What do you think of `userId` for the name?

Reply:
@samisayegh cannot change the name, that come from the SearchApi.